src/components/TopBar.js

```diff
@@ -14,10 +14,11 @@ import { Component } from 'src/libs/wrapped-components'
 
 const styles = {
   topBar: {
-    flex: 'none', height: 80,
+    flex: 'none', height: 100,
```

Comment:
TOL: Do we really want to add more vertical height here, given that users have been concerned with the lack of screen real estate?